<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>titlebar</class>
 <widget class="QWidget" name="titlebar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>20</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="closeButton" native="true">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>0</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QLabel" name="closeLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;x&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="minimizeButton" native="true">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>0</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QLabel" name="minimizeLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;-&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="maximizeButton" native="true">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>0</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QLabel" name="maximizeLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;+&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>2</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Stabilis&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
